visidata/loaders/xlsx.py

```diff
@@ -5,7 +5,7 @@ class XlsxIndexSheet(IndexSheet):
     'Load XLSX file (in Excel Open XML format).'
     rowtype = 'sheets'  # rowdef: xlsxSheet
     columns = [
-        Column('sheet', getter=lambda col,row: row.source.title),  # xlsx sheet title
+        Column('sheet', getter=lambda col,row: row.source.title or row.source.name),  # xlsx sheet title
```

Comment:
Is the title a unique aspect of the Excel data format? Is there a relationship between titles and sheet names? Can we just use sheet names?

Reply:
So as far as I tested `source.title` takes the _Excel sheet name_ from the tab at the bottom of the window. I think it should be kept as-is (without the `or`, as per my recent commit) because this aligns more closely with what is displayed in Excel.

Also `sheet.name` should probably use this logic as well, and if it does, we may as well do away with the `'sheet'` column altogether and show the `'name'` column?

Reply:
I think it is important to keep the `sheet` column for now because `IndexSheet`s contain rows that represent sheets.

It is also important for every sheet to have a name. Things break in VisiData (e.g. cmdlog) if there are un-named sheets!

Reply:
So, I think you should return the `title or name`.